tests/rules/test_python_module_error.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from thefuck.rules.python_module_error import match, get_new_command
+from thefuck.types import Command
+from thefuck.shells import shell
+
+
+@pytest.mark.parametrize(
+    "command",
+    [
+        Command(
+            "python some_script.py",
+            """Traceback (most recent call last):
+  File "some_script.py", line 1, in <module>
+    import more_itertools
+ModuleNotFoundError: No module named 'more_itertools'""",
+        ),
+        Command(
+            "./some_other_script.py",
+            """Traceback (most recent call last):
+  File "some_other_script.py", line 1, in <module>
+    from a_module import a_function
+ModuleNotFoundError: No module named 'a_module'""",
```

Comment:
Nit: How about defining a fixture and reusing it with different parameters?

Reply:
Ya, I thought about that, will do it right away.

Reply:
I meant an actual fixture:
```python
@pytest.fixture
def output(module):
    return """Traceback (most recent call last):
  File "some_script.py", line 1, in <module>
    import {0}
ModuleNotFoundError: No module named '{0}'""".format(
        module
    )


@pytest.mark.parametrize(
    "script, module",
    [
        ("python some_script.py", "more_itertools"),
        ("./some_other_script.py", "a_module"),
    ],
)
def test_match(script, output):
    assert match(Command(script, output))
```
Or even more:
```python
@pytest.mark.parametrize(
    "script, output",
    [
        ("python hello_world.py", "Hello World"),
        (
            "./hello_world.py",
            """Traceback (most recent call last):
  File "hello_world.py", line 1, in <module>
    pritn("Hello World")
NameError: name 'pritn' is not defined""",
        ),
    ],
)
def test_not_match(script, output):
    assert not match(Command(script, output))


@pytest.mark.parametrize(
    "script, module",
    [
        ("python some_script.py", "more_itertools"),
        ("./some_other_script.py", "a_module"),
    ],
)
def test_get_new_command(script, module, output):
    expected = "pip install {} && {}".format(module, script)
    assert get_new_command(Command(script, output)) == expected
```

Reply:
Hey, I updated my approach, with a fixture like function, but not actually a `pytest.fixture`, I think that my approach is more readable, understandable, and composable. Could you check it out and tell me how making a pytest.fixture would be a better approach, or if my approach is fine?

Reply:
The good thing about a fixture is that it lets you focus on the test inputs/output, and the closer they're from the actual test the better. Also, it helps in reducing the repetition of things. In the contrary, I find the one using fixtures and parametrizing only data, much more readable and concise. Also, there's an autoused fixture that lets you use `&&` regardlessly of the shell, (so no need for importing and using `and_`). Pytest fixtures FTW.

Reply:
Updated the tests!
How about now?